tools/_python_framework/Readme.md

```diff
@@ -33,14 +26,36 @@ sudo python mooltipass_tool.py
 - Select the 'vid:16d0 pid:09a0 rev:0100 mi:00' device, click 'Install'
 - Close the window
 
-## Arch Linux
+## Installation on Linux
+
+To access the device as normal user make sure you have set up the [udev rules correct](https://www.themooltipass.com/udev_rule.txt).
```

Comment:
what do you mean

On Sun, Aug 28, 2016 at 3:20 PM, Nico <notifications@github.com> wrote:

> In tools/_python_framework/Readme.md
> <https://github.com/limpkin/mooltipass/pull/234#discussion_r76531846>:
>
> > @@ -33,14 +26,36 @@ sudo python mooltipass_tool.py
> >  - Select the 'vid:16d0 pid:09a0 rev:0100 mi:00' device, click 'Install'
> >  - Close the window
> >
> > -## Arch Linux
> > +## Installation on Linux
> > +
> > +To access the device as normal user make sure you have set up the [udev rules correct](https://www.themooltipass.com/udev_rule.txt).
>
> Done. What about the crypto dependency?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/limpkin/mooltipass/pull/234/files/9818db561eab02d962b283236d3ef8f740914d36#r76531846>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF4ms2QSLDbjbM7NpZqVI9xPEeez3-kCks5qkYsPgaJpZM4Ju6_E>
> .
>